docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-summary-box/index.scss

```diff
@@ -0,0 +1 @@
+@forward "usa-summary-box";
```

Comment:
This needs to changes from the DS to reduce the heading size. See https://github.com/NCIOCPL/ncids/issues/1815 - you know what to do to comment the work around and the CGDP ticket to update.